Include/cpython/pystate.h

```diff
@@ -164,6 +164,8 @@ struct _ts {
     PyObject *async_gen_firstiter;
     PyObject *async_gen_finalizer;
 
+    PyObject *asyncio_cached_running_loop; // Strong reference
```

Comment:
`asyncio_running_loop` might be a better name (it's not really a cache anymore).